tsconfig.json

```diff
@@ -20,8 +28,11 @@
     "strictNullChecks": true,
     "target": "es2015"
   },
+  "exclude": [
+    "node_modules/@types/angular-mocks/node_modules/@types"
```

Comment:
I would like to once again register my general dislike of the current TS ecosystem. I think it means well and is still fixable. But I kinda hate it in its current form.